spec/factories/tickets.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :ticket do
+    total_fare { Faker::Number.between(from: 1, to: 1000) }
+
+    user { create(:user) }
+    trip { create(:trip) }
+    bus { create(:bus) }
+    payment { create(:payment) }
+    boarding { create(:boarding) }
```

Comment:
Can you please go through this reference: https://stackoverflow.com/questions/39459610/how-to-use-factorygirl-to-associate-user-id
only user, trip should do the trick here, as these are used to show the associations. Thanks!